src/routes/SQLScheduler/SQLScheduler.tsx

```diff
@@ -24,19 +26,16 @@ function SQLScheduler() {
 
   return (
     <div className="flex flex-col gap-2 h-full">
-      <h1 className="text-2xl">SQL Scheduler</h1>
-      <Text>Run SQL queries at regular intervals.</Text>
+      {!inCloudUi && (
```

Comment:
I think we should not make gc-admin aware of cloud ui, maybe some other setting would be nicer (i.e. `heading`).

Reply:
Yeah actually before adding `isCloudUi` in the context, there was a props in the `SQLScheduler` component to hide the headings, but then I have think about that we need to pass this property to all the components that will be used in the Cloud UI.
We can think about it when and if we have such a problem, happy to put the prop back 😄

Reply:
We can perhaps make it some sort of a context setting or something, but I would not make this aware at all that it can be run in cloud :)